src/App.js

```diff
@@ -1,19 +1,21 @@
 import { Route,Routes } from "react-router-dom";
 import "./App.css";
-import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage, VideoPage } from "./pages/index";
+import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage, VideoPage, Playlist, PlaylistPage } from "./pages/index";
 
 function App() {
   return (
     <>
-      <Routes>
+     <Routes>
      <Route path="/" element={ <Home/> }></Route> 
      <Route path="/signup" element={<LoginPage />} ></Route>
      <Route path="/history" element={ <History />} ></Route>
      <Route path="/watchlater" element={<WatchLater />} ></Route>
-     <Route path="/playlists" element={<WatchLater />} ></Route>
-     <Route path="/explore" element={<Videolist />} ></Route>
+     <Route path="/playlists" element={<Playlist />} ></Route>
+     <Route path="/playlists/:playlistId" element={<PlaylistPage />} ></Route>
+     {/* <Route path="/explore" element={<Videolist />} ></Route> */}
```

Comment:
Can remove this commented code

Reply:
I might add this later if required. Will remove later.